cloudify_cli/config/parser_config.py

```diff
@@ -448,7 +449,7 @@ def parser_config():
                         'arguments': {
                             '-b,--blueprint-id': blueprint_id_argument(),
                             '-o,--output': {
-                                'help': 'The output file path of the blueprint to be downloaded',
+                                'help': 'The local path of the downloaded blueprint',
```

Comment:
`cfy plugins download`'s `--output` argument states:
'A local path for the downloaded plugin',
and here `cfy blueprints download`'s `--output` states:
'**The** local path **of** the downloaded blueprint